.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
+
+[*.js]
+end_of_line = lf
+indent_style = space
+indent_size = 2
```

Comment:
this is great, thanks for adding it! Could you perhaps add `insert_final_newline = true`?